[CI] ScoreTests testSerializationOfSimple {TestCase=<boolean>} failing

**Build Scans:**
- [elasticsearch-periodic #9384 / release-tests](https://gradle-enterprise.elastic.co/s/qlpzrvm3atqnk)
- [elasticsearch-pull-request #81067 / checkpart3_release-tests](https://gradle-enterprise.elastic.co/s/vajuvg4wbacia)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:test" --tests "org.elasticsearch.xpack.esql.expression.function.fulltext.ScoreTests.testLiteralExpressions {TestCase=<boolean>}" -Dtests.seed=6D52917543CFDE20 -Dbuild.snapshot=false -Dtests.jvm.argline="-Dbuild.snapshot=false" -Dlicense.key=x-pack/license-tools/src/test/resources/public.key -Dtests.locale=es-SV -Dtests.timezone=Asia/Manila -Druntime.java=24
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.expression.function.fulltext.ScoreTests),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('testSerializationOfSimple%20%7BTestCase%3D%3Cboolean%3E%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Unknown NamedWriteable [org.elasticsearch.xpack.esql.core.expression.Expression][score]
```

**Issue Reasons:**
- [main] 2 failures in test testSerializationOfSimple {TestCase=<boolean>} (1.6% fail rate in 124 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -141,9 +141,6 @@ tests:
 - class: org.elasticsearch.packaging.test.DockerTests
   method: test090SecurityCliPackaging
   issue: https://github.com/elastic/elasticsearch/issues/131107
-- class: org.elasticsearch.xpack.esql.expression.function.fulltext.ScoreTests
-  method: testSerializationOfSimple {TestCase=<boolean>}
-  issue: https://github.com/elastic/elasticsearch/issues/131334
 - class: org.elasticsearch.packaging.test.DockerTests
   method: test071BindMountCustomPathWithDifferentUID
   issue: https://github.com/elastic/elasticsearch/issues/120917

